pkg/kubelet/pleg/generic.go

```diff
@@ -405,12 +405,18 @@ func (g *GenericPLEG) updateCache(pod *kubecontainer.Pod, pid types.UID) error {
 	// GetPodStatus(pod *kubecontainer.Pod) so that Docker can avoid listing
 	// all containers again.
 	status, err := g.runtime.GetPodStatus(pod.ID, pod.Name, pod.Namespace)
-	if klog.V(6).Enabled() {
-		klog.V(6).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name), "podStatus", status)
+	if err != nil {
+		if klog.V(6).Enabled() {
+			klog.V(6).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name), "podStatus", status)
```

Comment:
not related to this PR, I wonder if this file needs to clean up how it logs the pod. Sometimes it's a `podUID`, sometimes `podID`, sometimes `Name` and `Namespace` combination.

Reply:
Yes, the logs look somewhat inconsistent.

`klog.InfoS("SOME LOGS", "pod", klog.KObj(pod), "podUID", pod.UID)` looks good to me.

Reply:
will `klog.KObj(pod)` work? This is different pod type. I'm just not sure how KObj works.

Reply:
> not related to this PR, I wonder if this file needs to clean up how it logs the pod. Sometimes it's a `podUID`, sometimes `podID`, sometimes `Name` and `Namespace` combination.

Align kubelet log for podID and Namespace.